src/server.jsx

```diff
@@ -32,7 +34,16 @@ export default locals => {
           <meta name="theme-color" content="#2B3A42" />
           <meta name="viewport" content="width=device-width, initial-scale=1" />
           <title>{title}</title>
-          <meta name="description" content="webpack is a module bundler. Its main purpose is to bundle JavaScript files for usage in a browser, yet it is also capable of transforming, bundling, or packaging just about any resource or asset." />
+          <meta name="description" content={ description } />
+          <meta property="og:site_name" content="webpack" />
+          <meta property="og:type" content="website" />
+          <meta property="og:title" content={ title } />
+          <meta property="og:description" name="description" content={ description } />
+          <meta property="og:image" content={ Logo } />
+          <meta property="twitter:card" content="summary" />
+          <meta property="twitter:site" content="@webpack" />
+          <meta property="twitter:creator" content="@webpack" />
+          <meta property="twitter:domain" content="https://webpack.js.org/" />
```

Comment:
I am 99% sure that when sharing the link server will return static HTML.
I mean, there is not need for a React component for this.

Reply:
i am not sure what you mean, which react component?

Reply:
From the client side.
I mean, it is fine :)

Reply:
ah i thought it was change request, didnt see if thats approved or not :D